platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/files/DataFileWriter.java

```diff
@@ -62,6 +60,22 @@ public final class DataFileWriter {
      */
     private long dataItemCount = 0;
 
+    private final long dataBufferSize;
+
+    /**
+     * Create a new data file with moving mapped byte buffer of 256Mb size.
+     */
+    public DataFileWriter(
+            final String filePrefix,
+            final Path dataFileDir,
+            final int index,
+            final Instant creationTime,
+            final int compactionLevel)
+            throws IOException {
+        // TODO maybe we can use 128Mb buffer size (see DataFileWriterBenchmark)
+        this(filePrefix, dataFileDir, index, creationTime, compactionLevel, PAGE_SIZE * KIBIBYTES_TO_BYTES * 64);
```

Comment:
Should `PAGE_SIZE * KIBIBYTES_TO_BYTES * 64` be extracted to a static field like `DEFAULT_BUF_SIZE` with a short comment?